documentation/src/main/asciidoc/quickstart/guides/obtaining.adoc

```diff
@@ -14,6 +14,7 @@ hibernate-osgi:: Hibernate support for running in OSGi containers.
 hibernate-c3p0:: Integrates the link:$$http://www.mchange.com/projects/c3p0/$$[C3P0] connection pooling library into Hibernate
 hibernate-hikaricp:: Integrates the link:$$http://brettwooldridge.github.io/HikariCP/$$[HikariCP] connection pooling library into Hibernate
 hibernate-proxool:: Integrates the link:$$http://proxool.sourceforge.net/$$[Proxool] connection pooling library into Hibernate
+hibernate-jcache:: Integrates the link:$$https://jcp.org/en/jsr/detail?id=107$$[JCache] caching specification into Hibernate as a second-level cache provider.
```

Comment:
That sounds perfect

On Thu, May 26, 2016 at 9:25 AM Louis Jacomet <notifications@github.com>
wrote:

> In documentation/src/main/asciidoc/quickstart/guides/obtaining.adoc
> <https://github.com/hibernate/hibernate-orm/pull/1386#discussion_r64754566>
> :
>
> > @@ -14,6 +14,7 @@ hibernate-osgi:: Hibernate support for running in OSGi containers.
> >  hibernate-c3p0:: Integrates the link:$$http://www.mchange.com/projects/c3p0/$$[C3P0] connection pooling library into Hibernate
> >  hibernate-hikaricp:: Integrates the link:$$http://brettwooldridge.github.io/HikariCP/$$[HikariCP] connection pooling library into Hibernate
> >  hibernate-proxool:: Integrates the link:$$http://proxool.sourceforge.net/$$[Proxool] connection pooling library into Hibernate
> > +hibernate-jcache:: Integrates the link:$$https://jcp.org/en/jsr/detail?id=107$$[JCache] caching specification into Hibernate as a second-level cache provider.
>
> Something like:
>
> Integrates the JCache caching specification into Hibernate, enabling any
> compliant implementation to become a second-level cache provider.
>
> or even longer?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/hibernate/hibernate-orm/pull/1386/files/57d09dd939a7a1b6abd4faa09f3e32908401c76f#r64754566>
>